### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by bsnchan** [knative/community#492] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n3wscott** [comment]
oh snap! how does this work?!

**Reply by julz** [reply]
Dammit I came here to post "first" 😇

**Reply by bsnchan** [reply]
Introduce yourself!

**Reply by n3wscott** [reply]
ME? Hi I am Scott, aka n3wscott, aka "new scott". I have been working on Knative for a long time now. I work on some of the reconciler frameworks we use, the CI systems we have, and sources/eventing. 👋

**Comment by omerbensaadon** [comment]
This is another comment on this thread!!! 

I am Omer Bensaadon and I (and others) am currently working on making Knative more approachable and user-centered :)

**Comment by julz** [comment]
Hello I am @julz, I worked for a long time on Cloud Foundry and my favourite bits of Knative are the bits that make Kubernetes easier and less yamly.
